ci-operator/config/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main.yaml

```diff
@@ -0,0 +1,39 @@
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+releases:
+  initial:
+    integration:
+      name: "4.15"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.15"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tests:
+- as: e2e
+  run_if_changed: ^.github/workflows/push.yaml$|^server|^operator|^ci
+  steps:
+    cluster_profile: aws-rhtap-qe
+    test:
+    - as: e2e
+      cli: latest
+      commands: |
+        export OPERATOR_IMAGE="quay.io/konflux-workspaces/workspaces-operator:pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}"
+        export SERVER_IMAGE="quay.io/konflux-workspaces/workspaces-server:pr-${PULL_NUMBER}-${PULL_PULL_SHA:0:8}"
+        make -C e2e deploy-operator deploy-server test
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: ipi-aws
```

Comment:
Using cluster_claim to get a cluster would remove the need for using this workflow.